src/app/todo-list/todo-list.component.scss

```diff
@@ -0,0 +1,19 @@
+.todo-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+
+  .todo-item {
+    min-width: 500px;
+  }
+
+  .completed {
+    text-decoration: line-through;
+    color: green;
```

Comment:
To be 100% correct, you should have a variables.scss file, where you would store all colors, specific sizes, etc. and use a variable here